src/components/modals/Prompt.jsx

```diff
@@ -30,6 +31,11 @@ Prompt.propTypes = {
   secondaryButton: PropTypes.any.isRequired,
   /** Controls whether modal (and backdrop) are visible or not */
   isOpen: PropTypes.bool,
+  /**
+   * Callback that user can pass in, to be conditionally fired when
+   * user attempts to close modal. Also controls modal x
```

Comment:
> Also controls modal x

This tripped me up for a sec - can we say...

> When defined, the modal close button appears

Reply:
certainly 👍 I updated that in both Prompt and Dialog as well